composer.json

```diff
@@ -53,11 +53,11 @@
       "@lint",
       "@sniff",
       "@security",
-      "@test"
+      "@coverage"
     ],
     "coverage": [
-      "git diff origin/master... -- > diff.txt",
-      "diffFilter --phpunit diff.txt build/logs/clover.xml 80"
```

Comment:
I had removed this with #740, wasn't aware this was used here.

However, I believe that a service that collects the code coverage changes over time and displays the diff in the pull request is more useful.

What do you think?